features/LIBRARY.feature

```diff
@@ -0,0 +1,93 @@
+Feature: LIBRARY
+  Scenario: Editing books in the table
+    Given We are logged in as admin and we are in the LIBRARY section
+    When We click the edit button
+    And We change the title and / or author
```

Comment:
Musisz to zdanie rozdzielić te testy to konkretne kroki, nie możesz zostawić wyboru użytkownikowi. Czyli proponuje And Admin change the title i pod spodem And Admin change the author file